container/openejb-loader/src/main/java/org/apache/openejb/loader/FileUtils.java

```diff
@@ -24,14 +24,20 @@
 @SuppressWarnings("PMD.OverrideBothEqualsAndHashcode")
 public class FileUtils {
 
-    private static final java.util.Random _random = new java.util.Random();
+    private static final java.util.Random RANDOM = new java.util.Random();
```

Comment:
That is not a static type. to be `RANDOM`
Also, could check if it makes sense to update to [ThreadLocalRandom](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/ThreadLocalRandom.html )

Reply:
Okej I will check